New package: Boltz1 v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: Boltz1
- Repository: https://github.com/MurrellGroup/Boltz1.jl
- Created by: @AntonOresten
- Version: v0.1.0
- Commit: 0f29f43a3d213f7071754bacfd64609bfc6bfd28
- Reviewed by: @AntonOresten
- Reference: https://github.com/MurrellGroup/Boltz1.jl/commit/0f29f43a3d213f7071754bacfd64609bfc6bfd28#commitcomment-156928582
- Description: Julia bindings for Boltz-1
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
- Add exported `predict` function.
- Add public `Boltz1.Schema` submodule.
`````
<!-- END RELEASE NOTES -->

<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6182f50ef05d0d15001eabfe10007324cc4744937804e9d43938516df4263d1e769642cb8b147f2f8fd10da9540f22048a7d082114011dddc1b41f54212a151b808157897ecc090dc365091680a0c69ffd98f06b054e7cb00c3205926f6e970e4ed6e62d449ea06e9d1746458f05e01406f19e2ab3d820e2a5305df77497d3b32ab6d508ca6770f2b7f38532379ee68c8041871b02fb4c242afca3716c40c685b7c91c3f554d36b76554faa380769ffaf -->

AntonOresten commented:
[noblock] Automerge should fail since [Boltz.jl](https://github.com/LuxDL/Boltz.jl) is already registered, with the levenshtein distance between them being <=2. The name 'Boltz1.jl' has been [approved by Avik](https://github.com/MurrellGroup/Boltz1.jl/issues/3#issuecomment-2876772641).

Tests are currently failing on GitHub's CI since it only has CPU, and the last registered version of [boltz](https://github.com/jwohlwend/boltz) seems to generally fail on CPU, but the but the package still loads and runs fine on GPU:

![image](https://github.com/user-attachments/assets/bcf4819f-1e17-4401-8293-5ae21ff6207c)

See https://github.com/MurrellGroup/Boltz1.jl/issues/2 for an issue tracking this, with a link to the Python repo issue. It seems it's fixed on main, but they haven't yet registered a new version on PyPI. I could potentially point the version to their main, but it might not be stable. For GPU testing I could also ask the JuliaGPU folks for CI servers with GPUs.

Either way, because of the name, I plan to drop a message in the `#pkg-registration` Slack channel in 3 days.

JuliaRegistrator committed:
New package: Boltz1 v0.1.0

UUID: 0bca4874-a2b8-4dca-bc03-d2d86e344f1f
Repo: https://github.com/MurrellGroup/Boltz1.jl.git
Tree: 1e8a78179f7be98206b869052ca6b9c01d38dfef

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

AntonOresten commented:
new PR: #130900 
see https://julialang.slack.com/archives/C6M4DQA5P/p1747166341838319

goerz commented:
Closing in favor of https://github.com/JuliaRegistries/General/pull/130900
